src/plugin.js

```diff
@@ -171,6 +174,26 @@ export const setupSessions = (player) => {
   }
 };
 
+/**
+ * Construct a simple function that can be used to dispatch EME errors on the
+ * player directly, such as providing it to a `.catch()`.
+ *
+ * @function emeErrorHandler
+ * @param    {Player} player
+ * @return   {Function}
+ */
+export const emeErrorHandler = (player) => {
+  return (objOrErr) => {
+    const message = objOrErr ? objOrErr.message || objOrErr : undefined;
```

Comment:
Probably better to use `void 0`.

Reply:
Why's that? Do people actually reassign `undefined`? :scream:

Also, I thought that was something babel did? I figured babel was the reason for all the `void 0` in the transpiled video.js?

Reply:
Using undefined directly isn't 100% safe because you can re-assign it.
Can/should `null` be used?

Reply:
Tested that passing `null` for the message uses the default for the code, and it does, so I changed it to `null`.